tests/fixtures.py

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
 
-from copy import deepcopy
-
 import pytest
-
+from sys import version_info
 from pydash._compat import iteritems
 
+if version_info[0] > 2:
```

Comment:
I'd like to avoid having to modify the pytest arguments in `tasks.py` (i.e. by changing test targets from `'tests'` to `'tests/test_*'`) and relying on conditionally importing the test function into a non-"test_" prefixed file (`fixtures.py`).

Since the annotations test is very minor, I'd prefer to add it in a way that requires the fewest changes even if the solution is a hack (which it will be :smile:):

```python
# test_annotations.py

import pytest

import pydash as _
from pydash._compat import PY3


if PY3:
    # Hack around the fact that we can't define this kind of function in
    # Python 2 so have to rely on conditionally creating it using exec()
    # to avoid syntax errors while still having this test case covered for
    # Python 3.
    exec('def typed_function(row: int, index: int, col: list): return row + 1')
else:
    typed_function = None


@pytest.mark.skipif(not PY3, reason='test requires Python 3 annotations')
def test_annotated_callback():
    assert _.map_([1, 2], typed_function) == [2, 3]
```

It's not pretty but I think it'll do for now.